<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Android Service Uninstaller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>461</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="dwn_btn">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="con_btn">
       <property name="text">
        <string>Connection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_btn">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="con_output">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
